amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/BlockOnParentShardTask.java

```diff
@@ -54,28 +54,29 @@
     @Override
     public TaskResult call() {
         Exception exception = null;
-
+        final String shardInfoId = shardInfo.streamIdentifierSerOpt().map(s -> s + ":" + shardInfo.shardId())
```

Comment:
May be too late. Can we have constant for the delimiter so it is easier to change if needs to be changed ?  Can we move this to multistream config and be applied only to multi stream

Reply:
This is used just for logging. Does not impact any serde logic.